rust-toolchain.toml

```diff
@@ -1,5 +1,4 @@
 [toolchain]
-channel = "nightly-2024-02-04"
```

Comment:
Maybe you want to keep the channel entry and set it to:
```
channel = "stable-2022-11-03"
```
This should bring the 1.65.0 version to the game. I've found it's release date here: https://releases.rs/docs/1.65.0/. 

Indeed:
```
$ nix develop
...
$ rustc --version
rustc 1.65.0 (897e37553 2022-11-02)
```